rusk-wallet/src/main.rs

```diff
@@ -247,64 +242,67 @@ async fn main() -> Result<(), Error> {
 async fn exec() -> Result<(), Error> {
     use CliCommand::*;
 
-    // parse cli arguments
-    let cfg: WalletCfg = WalletCfg::parse();
+    // load configuration (or use default)
+    let mut cfg = Config::load().unwrap_or_else(Config::default);
+
+    // parse user args and merge with static config
+    let args = WalletArgs::parse();
+    let cmd = args.command.clone();
+    cfg.merge(args);
 
     // create directories
-    let data_dir = cfg.data_dir.unwrap_or_else(LocalStore::default_data_dir);
-    LocalStore::create_dir(&data_dir)?;
+    LocalStore::create_dir(&cfg.wallet.data_dir)?;
 
     // get command or default to interactive mode
-    let cmd = cfg.command.unwrap_or(CliCommand::Interactive);
+    let cmd = cmd.unwrap_or(CliCommand::Interactive);
 
     // request auth for wallet (if required)
-    let pwd = if cmd.uses_wallet() || cfg.wallet_file.is_some() {
+    let pwd = if cmd.uses_wallet() || cfg.wallet.file.is_some() {
         prompt::request_auth("Please enter wallet password")
     } else {
         blake3::hash("".as_bytes())
```

Comment:
Unrelated to this PR, but seems this should be a constant

Reply:
Don't you think this is verbose enough? I think a constant for an empty string distracts more than helps.

Reply:
Oh, I mean a constant for the hash, not for the empty string. But its fine, its unrelated to this PR